core/src/main/java/io/kestra/core/models/tasks/retrys/AbstractRetry.java

```diff
@@ -33,6 +34,8 @@ public abstract class AbstractRetry {
     @Builder.Default
     private Boolean warningOnRetry = false;
 
+    public abstract Instant getNextDate(Integer attemptCount, Instant lastAttempt);
```

Comment:
```suggestion
    public abstract Instant nextRetryDate(Integer attemptCount, Instant lastAttempt);
```

Moreover, if you start with get it will be serialized.